### PR TITLE
Support setting link state for SR-IOV VFs

### PR DESCRIPTION
**Pull request by booxter**

### CODE REVIEW

**Comment by booxter** [comment]
@vishvananda @aboch please take a look. SR-IOV CNI and KubeVirt are waiting for this change to land so that we can set up PFs in CI environment for inter-connectivity testing. Thank you!

File: link_linux.go [hunk 1]
```diff
@@ -559,6 +559,36 @@ func (h *Handle) LinkSetVfRate(link Link, vf, minRate, maxRate int) error {
...
 
+// LinkSetVfState enables/disables virtual link state on a vf.
+// Equivalent to: `ip link set $link vf $vf state $state`
+func LinkSetVfState(link Link, vf int, state uint32) error {
+	return pkgHandle.LinkSetVfState(link, vf, state)
+}
+
+// LinkSetVfState enables/disables virtual link state on a vf.
+// Equivalent to: `ip link set $link vf $vf state $state`
+func (h *Handle) LinkSetVfState(link Link, vf int, state uint32) error {
```
Comment (fedepaol) [inline link_linux.go]: Would it make sense to have the state as a bool (down, up)?

Comment (fedepaol) [reply link_linux.go]: Nevermind, just saw that it's not yes / no

**Comment by aboch** [comment]
LGTM

**Pull request merged**